job_runner/cli.py

```diff
@@ -1,13 +1,14 @@
 import argparse
+import json
 import os
 import signal
 import sys
 import json
 
 import job_runner.reporting as reporting
+from job_runner import JOBS_FILE
```

Comment:
Not related to the commit text

Reply:
I think this was the doing of isort (which I know we don't use) sorting and cleaning unused imports